build.gradle

```diff
@@ -20,6 +20,7 @@ repositories {
 
 dependencies {
     compileOnly 'com.gradle:gradle-enterprise-gradle-plugin:3.13.1'
+    testImplementation 'junit:junit:4.13.2'
```

Comment:
```suggestion
    testImplementation(platform('org.junit:junit-bom:5.9.3'))
	testImplementation('org.junit.jupiter:junit-jupiter')
```

You may also need:

```
test {
	useJUnitPlatform()
}
```

Reply:
[9580449](https://github.com/gradle/common-custom-user-data-gradle-plugin/pull/169/commits/95804491c3badf954318461205fe63e77a96c119)